seatunnel-core/seatunnel-flink-starter/seatunnel-flink-starter-common/src/main/java/org/apache/seatunnel/core/starter/flink/execution/SinkExecuteProcessor.java

```diff
@@ -175,4 +194,40 @@ public SeaTunnelSink fallbackCreateSink(
         source.prepare(pluginConfig);
         return source;
     }
+
+    // catalogName.databaseName.[schemeName].tableName -> databaseName.[schemeName].tableName
+    public String extractTableIdName(TableIdentifier tableIdentifier) {
+        StringBuilder tableId = new StringBuilder();
+        if (tableIdentifier.getDatabaseName() != null) {
+            tableId.append(tableIdentifier.getDatabaseName());
+        }
+        if (tableIdentifier.getSchemaName() != null) {
+            if (tableIdentifier.getDatabaseName() != null) {
+                tableId.append(".");
+            }
+            tableId.append(tableIdentifier.getSchemaName());
+        }
+        if (tableIdentifier.getTableName() != null) {
+            if (tableId.length() != 0) {
+                tableId.append(".");
+            }
+            tableId.append(tableIdentifier.getTableName());
+        }
+        return tableId.toString();
+    }
```

Comment:
duplicate with the method in the seatunnel-core/seatunnel-flink-starter/seatunnel-flink-13-starter/src/main/java/org/apache/seatunnel/core/starter/flink/execution/SinkExecuteProcessor.java ?